bin/fetchl10nstrings.js

```diff
@@ -0,0 +1,351 @@
+#! /usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+/**
+ * Create the string bundles and then save them to disk.
+ */
+/*jshint node:true, esnext: true*/
+"use strict";
+const async = require("marcosc-async");
+const fs = require("fs");
+const path = require("path");
+const clc = require("cli-color");
+const fetch = require("node-fetch"); // jshint ignore:line
```

Comment:
jshint thinks I'm overriding the browser's fetch, hence this is needed. I should file a bug on jshint, but I'm too lazy.